src/Profiles/BicyclePower/ANTPLUS_BicyclePowerPrivateDefines.h

```diff
@@ -0,0 +1,119 @@
+#ifndef ANTPLUS_BICYCLEPOWERDEFINES_h
+#define ANTPLUS_BICYCLEPOWERDEFINES_h
+
+/* Channel Config */
+#define ANTPLUS_BICYCLEPOWER_DEVICETYPE												11

```

Comment:
in the future this will be public for device reference when searching

Reply:
this is a duplicate define, public defines should be included here so preprocessor doesn't get mad accidentally :)